.github/workflows/release.yaml

```diff
@@ -183,19 +182,24 @@ jobs:
         msg="Release ${{ env.RELEASE_VERSION }}"
         git tag --annotate --message "${msg}" ${{ env.RELEASE_VERSION }}
         # push both the tag as well as a release branch with that tag.
-        git push origin releases/${{ env.RELEASE_VERSION }} ${{ env.RELEASE_VERSION }}
+        git push origin ${{ env.RELEASE_VERSION }}
 
     - name: Create GPG Token file from Secret
       run: |
         echo "${{ secrets.GPG_PRIVATE_KEY_FOR_SIGNING }}" > ocm-releases-key.gpg
         echo "GPG_KEY_PATH=ocm-releases-key.gpg" >> $GITHUB_ENV
 
+    - name: Setup Release Notes as File to make it readable by GoReleaser
+      run: |
+        echo "${{ fromJSON(needs.check.outputs.draft-release-notes).body }}" > $RUNNER_TEMP/RELEASE_NOTES.md
+        echo "RELEASE_NOTES=$RUNNER_TEMP/RELEASE_NOTES.md" >> $GITHUB_ENV
+        cat $RUNNER_TEMP/RELEASE_NOTES.md
```

Comment:
debug leftover or intention?

Reply:
no i wanted to have the real release notes logged in the stepped. do you think thats a problem?